make/stratos/run

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env bash
+
+set -o errexit -o nounset
+
+. "${BASH_SOURCE%/*}/_env"
+. "${BASH_SOURCE%/*}/_has_release.sh"
+
+# TODO(f0rmiga): Refactor make/run and this file to move the following variables to a single source.
```

Comment:
…And this could turn into `make/include/defaults` or `…/variables` or something.